doc/releases/release-notes-3.7.rst

```diff
@@ -358,6 +360,8 @@ Bluetooth
   * Added support for Infineon CYW208XX.
   * Added support for Renesas SmartBond DA1469x.
   * Removed unmaintained B91 driver.
+  * Added support for NXP IW612 Chipset on boards mimxrt1170_evkb and mimxrt1040_evk.
+    It can be enabled by :kconfig:option:`CONFIG_BT_NXP_NW612`.
```

Comment:
Is this different from the "Added support for NXP platforms" a few lines earlier? If so, maybe that other line needs specifying exactly what (other) NXP platforms it's referring to?

Reply:
I think they are different. "Added support for NXP platforms" mainly refers to a platform where the host and controller are on the same chip. And what I want to describe is that a separate Bluetooth controller module is enabled in this release.
@dleach02 Is it correct?

Reply:
Have udpated.